plugins/outputs/nats/README.md

```diff
@@ -74,6 +74,13 @@ to use them.
     # name = ""
     # subjects = []
 
+    ## Use asynchronous publishing for higher throughput, but note that it does not guarantee order within batches.
+    # async_publish = false
+
```

Comment:
One question here: Do you also plan to add async publishing to the non-jetstream case? Otherwise the option name is misleading...

Reply:
@srebhan, NATS Core (non-jetstream) is already "async", it only have one publish method. NATS Core do not have publish ack. "It is publish and go on/forget"